releases_test.go

```diff
@@ -164,6 +164,9 @@ func findReleases(provider string, archived bool) ([]v1alpha1.Release, error) {
 func releasesToIndex(releases []v1alpha1.Release) []versionbundle.IndexRelease {
 	var indexReleases []versionbundle.IndexRelease
 	for _, release := range releases {
+		if release.Spec.State != "active" {
```

Comment:
Is this change still required? We have a newer aws-operator in the release.

Reply:
Reverted 👍🏻